packages/types/README.md

```diff
@@ -0,0 +1,26 @@
+[![](https://user-images.githubusercontent.com/39338561/122415864-8d6a7c00-cf88-11eb-846f-a98a936f88da.png)
+](https://kilt.io)
+
+![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)
+
+# KILT Types
+
+This package provides the generally applicable TypeScript types for different mechanisms in the KILT SDK.
```

Comment:
```suggestion
This package provides the generally applicable TypeScript types for different operations provided by the KILT SDK.
```

Reply:
Hmm, not sure if operations is enough here. I wanted to point out, that it contains all the base types you would also need to rebuild an SDK basically (while currently not knowing, if that is true)

Reply:
ah, but I say this in the next paragraph... so maybe it is fine to take your suggestion

Reply:
a05286eff11b655e0641df1ce8938f24b4bad710

Reply:
5fed91be8f69ca7a08bf7cad96755c93aec05f74